### ISSUE TITLE
Cannot upgrade from kube-up

### ISSUE THREAD

**Issue by aneeshd16** [kubernetes/kops#1320]
I am following the directions to convert my `kube-up` cluster to a kops one from here: https://github.com/kubernetes/kops/blob/master/docs/upgrade_from_kubeup.md

According to the first step,
```sh
export OLD_NAME=kubernetes
export REGION=ap-southeast-1
kops import cluster --region ${REGION} --name ${OLD_NAME}
```
(The `KubernetesCluster` tag value is `kubernetes`)
This gives me:

```
I0103 12:43:00.693037   24852 import_cluster.go:115] Found master: "i-0*****************"
W0103 12:43:03.295964   24852 import_cluster.go:268] Multiple Autoscaling groups found
I0103 12:43:03.546303   24852 vfs_castore.go:413] Issuing new certificate: "ca"

Cluster Name must be a fully-qualified DNS name (e.g. --name=mycluster.myzone.com)
```
And it exits after that. 

My old k8s cluster is running on v1.3

**Comment by hernandanielg** [comment]
I am having same issue, I tried changing KubernetesCluster tag on EC2 instance master node and adding an A record to my zone, but I have not success doing it. :(

**Comment by chrislovecnm** [comment]
Paging @justinsb :)  I know this is important since we want to remove kube-up.sh ...

**Comment by hernandanielg** [comment]
Since I am in a test environment I re-created the cluster using a FQDN as a CLUSTER_ID using:   
`export KUBE_AWS_INSTANCE_PREFIX=k8s.example.domain`

Now the problem is different:
```
➜  ~  kops import cluster --region us-east-1 --name **** --state ****
I0104 16:31:00.474274   15285 import_cluster.go:115] Found master: "i-****"
W0104 16:31:01.508197   15285 import_cluster.go:268] Multiple Autoscaling groups found
I0104 16:31:01.689359   15285 vfs_castore.go:413] Issuing new certificate: "ca"

Cluster.Spec.KubeControllerManager.CloudProvider: Invalid value: "": Did not match cluster CloudProvider
```
Any help? Maybe I can set the CloudProvider in kubelet but it seems this procedure has a lot of bugs.

**Comment by justinsb** [comment]
I validated this with beta1, and did find a few errors which were introduced over 1.5.0 development.  I tested that they will be fixed by #1699 (I tested with 1.2 and 1.3).

**Comment by justinsb** [comment]
This should have been fixed by #1699, so kops 1.5.0 (coming out today hopefully) should fix it!
